### PR TITLE
Editorial: clean up Dependencies section

### PR DESCRIPTION
**Pull request by Honry**
Split from https://github.com/w3c/remote-playback/pull/133

***
<a href="https://pr-preview.s3.amazonaws.com/Honry/remote-playback/pull/134.html" title="Last updated on Mar 19, 2020, 5:54 AM UTC (0117017)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/remote-playback/134/2fe00fb...Honry:0117017.html" title="Last updated on Mar 19, 2020, 5:54 AM UTC (0117017)">Diff</a>

### CODE REVIEW

**Comment by Honry** [comment]
@mounirlamouri, this is the first PR to clean up the Dependencies section. PTAL, thanks!

File: index.html [hunk 1]
```diff
@@ -672,47 +549,49 @@ <h5>
...
+              for a <a data-cite="HTML#media-resource">media resource</a>
+              if the <a>user agent</a> can reasonably guarantee that the
+              remote playback of the media specified by the resource will
+              succeed on that device.
             </p>
             <p class="note">
-              The <a>user agent</a> can use the <a><code>srclang</code></a>
-              attribute of the <a><code>track</code></a> element as a hint of
+              The <a>user agent</a> can use the <code>{{HTMLTrackElement/srclang}}</code>
+              attribute of the <code>[^track^]</code> element as a hint of
```
Comment (mounirlamouri) [inline index.html:569]: Are those changes from elements to markdown required? (here and other places in this pull request)

Comment (Honry) [reply index.html:569]: No, but it's a more simplified way ReSpec provided, you can search your references via https://respec.org/xref/?term=track.

**Review by mounirlamouri — APPROVED** [review]

**Comment by Honry** [comment]
Thanks @mounirlamouri, could you merge it?

**Comment by anssiko** [comment]
Thanks @Honry!

**Pull request merged**